findimports.py

```diff
@@ -97,6 +99,86 @@
+    "nt", "_pickle", "platform", "traceback",
+    "_json", "numbers", "_dbm", "_stat",
+    "pydoc", "site", "_overlapped", "_osx_support",
+    "_sha512", "tokenize", "queue", "contextlib",
+    "tarfile", "cgitb", "ntpath", "timeit",
+    "webbrowser", "antigravity", "_functools", "uu",
+    "functools", "imp", "_contextvars", "ftplib",
+    "importlib", "zipapp", "bz2", "inspect",
+    "rlcompleter", "bdb", "runpy", "typing",
+    "_codecs", "errno", "mailcap", "graphlib",
+    "reprlib", "_sha3", "imghdr", "msvcrt",
+    "turtle", "ast", "difflib", "shlex",
+    "pwd", "posixpath", "heapq", "pstats",
+    "unittest", "_tkinter", "_multiprocessing", "plistlib",
+    "smtpd", "filecmp", "netrc", "codecs",
+    "crypt", "getopt", "os", "_curses_panel",
+    "_pyio", "glob", "winreg", "_socket",
+    "argparse", "io", "weakref", "subprocess",
+    "_gdbm", "pdb", "sre_compile", "_ast",
+    "threading", "tracemalloc", "cProfile", "doctest",
+    "gzip", "opcode", "_aix_support", "dataclasses",
+    "py_compile", "linecache", "stringprep", "uuid",
+    "copyreg", "smtplib", "sre_constants", "_ssl",
+    "pathlib", "sunau", "_bootsubprocess", "tabnanny",
+    "sndhdr", "aifc", "secrets", "unicodedata",
+    "_elementtree", "termios", "_collections", "_heapq",
+    "_statistics", "_lzma", "_py_abc", "nntplib",
+    "_zoneinfo", "colorsys", "optparse", "pickletools",
+    "sys", "configparser", "_signal", "_warnings",
+    "code", "sysconfig", "struct", "copy",
+    "faulthandler", "_crypt", "pyexpat", "nis",
+    "contextvars", "_random", "_weakrefset", "http",
+    "time", "_imp", "codeop", "binhex",
+    "encodings", "ensurepip", "select", "collections",
+    "_sha1", "_codecs_cn", "_threading_local", "_weakref",
+    "sqlite3", "_symtable", "pyclbr", "tkinter",
+    "telnetlib", "_frozen_importlib", "_codecs_tw", "_io",
+    "_opcode", "_winapi", "_string", "builtins",
+    "signal", "ipaddress", "xml", "operator",
+    "compileall", "_hashlib", "distutils", "_compression",
+    "cmath", "warnings", "nturl2path", "_asyncio",
+    "json", "posix", "pipes", "pprint",
+    "stat", "lzma", "_queue", "_bisect",
+    "__future__", "ssl", "_abc", "_thread",
+    "hmac", "gc", "_lsprof", "_compat_pickle",
+    "math", "mimetypes", "tty", "zipimport",
+    "marshal", "socketserver", "_ctypes", "bisect",
+    "shutil", "_pydecimal", "ossaudiodev", "xmlrpc",
+    "asyncio", "multiprocessing", "_sha256", "string",
+    "email", "dis", "dbm", "pydoc_data",
+    "ctypes", "binascii", "zlib", "_sqlite3",
+    "_sitebuiltins", "keyword", "_locale", "mailbox",
+    "cmd", "hashlib", "spwd", "fcntl",
+    "sre_parse", "_posixsubprocess", "_md5", "fileinput",
+    "pkgutil", "random", "chunk", "_posixshmem",
+    "syslog", "types", "wsgiref", "asyncore",
+    "statistics", "_decimal", "_curses", "_tracemalloc",
+    "html", "shelve", "_datetime", "decimal",
+    "token", "trace", "abc", "audioop",
+    "_codecs_kr", "array",
```

Comment:
I'd sort this alphabetically, but I suppose it doesn't really matter.

As for my earlier mention of the Python 3.10 feature, I wasn't suggesting bumping the minimum required Python version, but maybe it's worth using it directly, if it exists, so that new stdlib modules appearing in future Python versions will automatically be recognized?  I.e. something like

```python
STDLIB_MODNAMES_SET = getattr(sys, 'stdlib_module_names', frozenset([
    # taken from Python 3.10
    '__future__', '_abc', ...
]))
```